src/pages/Edit.js

```diff
@@ -1,65 +1,37 @@
-import React,{useState,useEffect} from "react";
-import {useParams,useHistory} from 'react-router-dom'
-import {Form, Button,Col} from 'react-bootstrap'
+import React, { useState, useEffect } from "react";
```

Comment:
Actually, this page seems .. just copied and pasted? Not ready? This seems like a copy of something else.

Reply:
i will remove the Edit.js because we don't need it